File: 1_Two_Sum.py
# PROBLEM: Two Sum

class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """

        total_value = {} 

        for i, num in enumerate(nums):  
            value = target - num
            if value in total_value:
                return [total_value[value], i]

            total_value[num] = i

File: 1_TWO_SUM.java
// PROBLEM : (1) TWO SUM

// SOLUTION :

class Solution {
    public int[] twoSum(int[] nums, int target) {
       
       for(int i = 1 ; i < nums.length; i++){
        for(int j = 0 ; j <  nums.length - i; j++){
            if(nums[j] + nums[j+i] == target)
            {
                return new int[]{j,j + i};
            }
        }
       }
    return new int[]{};
    }
}
